huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go

```diff
@@ -292,6 +298,43 @@ func ResourceCluster() *schema.Resource {
 				Type:     schema.TypeBool,
 				Optional: true,
 			},
+			"component_congurations": {
+				Type:     schema.TypeList,
+				Optional: true,
+				ForceNew: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
```

Comment:
`name` and `congurations` should be required?

Reply:
make the `name` required